composer/algorithms/mixup/README.md

```diff
@@ -80,13 +124,17 @@ Doing so avoids putting additional work on the CPU (since augmentation occurs on
 > MixUp requires a small amount of additional GPU compute and memory to produce the mixed-up batch.
 > In our experiments, we have found these additional resource requirements to be negligible.
 
-> ❗ MixUp Produces a Full Distribution, Not a Target Index
+> ❗ When `interpolate_loss=False` MixUp Produces a Full Distribution, Not a Target Index
 >
 > Many classification tasks represent the target value using the index of the target value rather than the full one-hot encoding of the label value.
-> Since MixUp interpolates between two target values for each example, it must represent the final targets as a dense distribution.
+> With `interpolate_loss=False` MixUp interpolates between two target values for each example, it must represent the final targets as a dense distribution.
 > Our implementation of MixUp turns each label into a dense distribution (if it has not already been converted into a distribution).
 > The loss function used for the model must be able to accept this dense distribution as the target.
 
+>❗ When `interpolate_loss=True` MixUp interpolates the loss rather than the targets.
+
+> This is fine for loss functions that are linear in the targets, such as cross entropy, but may produce unexpected results for other loss functions.
```

Comment:
Currently shows up as two different blocks [in the docs](https://mosaicml-composer--825.com.readthedocs.build/en/825/method_cards/mixup.html).
```suggestion
>❗ When `interpolate_loss=True` MixUp interpolates the loss rather than the targets.
> 
> This is fine for loss functions that are linear in the targets, such as cross entropy, but may produce unexpected results for other loss functions.
```

Reply:
also, at the bottom of the method card, should "mixup" be capitalized in the paper name?

Reply:
Fixed the the blocking here. 

For the name, the original paper used "mixup" in all lower case, so for the citation I left it as it appears in the paper. Is that what you meant?

Reply:
yes; sgtm